_notebooks/textbook.py

```diff
@@ -102,3 +108,29 @@ def get_chapter_pdf_for_book(book: Book) -> List:
                 number=int(file.stem[-2:]),
             )
             self.chapters.append(chp)
+
+    def clean_raw_text(self, disable_pysbd=False):
+        """pySBD (Python Sentence Boundary Disambiguation)
```

Comment:
Excellent docstring, even if it's a little too verbose :P

Reply:
YAY